### PR TITLE
Mkfehmdir tests 1

### PR DESCRIPTION
**Pull request by dmwinslow**
First batch of test for utilities that support mkfehmdir. The strategy here is to test all functions that modify files in some way, but not test `mkfehmdir` as a whole - since the only added benefit there is testing Matlab's `copyfile` functionality, which is unnecessary.

I've broken out the first few utilities here (`mkdotfiles`, `datreset`, `modeltime`, `datcopy`), the rest will come in a separate PR.

No surprises on the outputs in this one, everything matches as expected!

### CODE REVIEW

File: matlab_archive/test/test_modeltime.m [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+%% modeltime np2d_cond
+test_against_expected('np2d_cond', 36500000000, 18024418.611298);
+
+
+%% modeltime np3d_cond
+test_against_expected('np3d_cond', 5475000000, 1338452148.437500);
+
+
+function test_against_expected(fixture_name, expected_time, expected_deltime)
```
Comment (dmwinslow) [inline matlab_archive/test/test_modeltime.m:11]: Note this test function is much simpler than the others. The fact that the function doesn't write anything means we don't need to make a test directory.

Comment (hydrocrust) [reply matlab_archive/test/test_modeltime.m:11]: Not sure of meaning for expected_time (for routine to run?) and then expected_deltime (difference from earlier version)? Maybe you can tell me when we speak next. Thanks!

**Comment by hydrocrust** [comment]
> First batch of test for utilities that support mkfehmdir. The strategy here is to test all functions that modify files in some way, but not test `mkfehmdir` as a whole - since the only added benefit there is testing Matlab's `copyfile` functionality, which is unnecessary.
> 
> I've broken out the first few utilities here (`mkdotfiles`, `datreset`, `modeltime`, `datcopy`), the rest will come in a separate PR.
> 
> No surprises on the outputs in this one, everything matches as expected!

Yay, good news. A

File: matlab_archive/test/test_modeltime.m (hunk 1 above)
Comment (dmwinslow) [reply matlab_archive/test/test_modeltime.m:11]: Good question - no these refer to the times that I'm reading from the .hist file - so in Myrs etc. The test ensures that the function reading the file and inferring the final timestep (deltime) are what I expect from having inspected the files.

You can also write tests to check things like runtime as well, but that's less common that testing functionality like this.

**Pull request merged**